src/diffpy/labpdfproc/tools.py

```diff
@@ -1,9 +1,40 @@
+import glob
+import os
 from pathlib import Path
 
 WAVELENGTHS = {"Mo": 0.71, "Ag": 0.59, "Cu": 1.54}
 known_sources = [key for key in WAVELENGTHS.keys()]
 
 
+def set_input_files(args):
+    """
+    Set input directory and files
+
+    Parameters
+    ----------
+    args argparse.Namespace
+        the arguments from the parser
+
+    Returns
+    -------
+    args argparse.Namespace
+
+    """
+    if not Path(args.input).exists():
+        raise ValueError("Please specify valid input file or directory.")
+
+    if not Path(args.input).is_dir():
+        input_dir = Path.cwd() / Path(args.input).parent
```

Comment:
I wonder whether this will fail if the user gives a filename with a path that doesn't include cwd?  For example, I think a valid test would be:
```
cd /user/me/analysis
labpdfcor 2.5 /user/me/data/my_file.xy
```
Please could you add test for this situation and make sure it passes?